ducktape/errors.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2015 Confluent Inc.
```

Comment:
1. Yes, but since we're just adding these, lets defer those to separate patches. There are bunch of places we raise generic exceptions where we shouldn't. There are also other exceptions I'd like to change (e.g., not expose subprocess.CalledProcessError directly, instead converting it to our own type; that's an implementation detail and could potentially change if, for example, we used paramiko).
2. Yes, that's a good idea. Have updated the patch to include DucktapeError.